pkg/apis/integreatly/v1alpha1/installation_types.go

```diff
@@ -35,9 +35,14 @@ type InstallationSpec struct {
 	// INSERT ADDITIONAL SPEC FIELDS - desired state of cluster
 	// Important: Run "operator-sdk generate k8s" to regenerate code after modifying this file
 	// Add custom validation using kubebuilder tags: https://book.kubebuilder.io/beyond_basics/generating_crd.html
-	Type            string `json:"type"`
-	NamespacePrefix string `json:"namespacePrefix"`
-	SelfSignedCerts bool   `json:"selfSignedCerts"`
+	Type            string                      `json:"type"`
+	NamespacePrefix string                      `json:"namespacePrefix"`
+	SelfSignedCerts bool                        `json:"selfSignedCerts"`
+	AMQOnlineConfig InstallationAMQOnlineConfig `json:"amqOnline"`
```

Comment:
@maleck13 Does this CR struct change seem ok to you? I've included the default set of resources we'd want to install in `resources/default-amqonline-resources.json`.